massgov/openmass
# DP-22747: Run Backstop on flexible org branch

**Description:**
Fixing some discrepancies found after running Backstop tests.


**Jira:** (Skip unless you are MA staff)
DP-22747


**To Test:**
- Go to /orgs/qag-executive-office-of-technology-services-and-security.
- Verify the feedback form is displayed at the bottom of the page.
- Verify any image credits are displayed at before the feedback form.
- Verify the governmentOrganization schema data is at the bottom of the page source.
- Verify there is no Upcoming Events section.
- Go to /orgs/qag-executive-office-of-technology-services-and-security/services.
- Verify you get a 404 error.
- Go to /orgs/qag-digital-services.
- Verify there is an “Upcoming events” section.


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
